.gitignore

```diff
@@ -105,3 +105,15 @@ venv.bak/
 
 # Mac OSX
 .DS_Store
+
+# Binary files
+*.pickle
+
+# Sphinx HTML Output
+html/
+
```

Comment:
I don't get the sphinx/sphinx gallery folder structure. Could you explain to me? 

Usually Sphinx had `docs/_build` folder where `_build` was already included in `.gitignore`.